v3/newrelic/code_level_metrics.go

```diff
@@ -142,6 +171,46 @@ func WithFunctionLocation(function interface{}) TraceOption {
 	}
 }
 
+//
+// WithDefaultFunctionLocation is like WithFunctionLocation but will only
+// evaluate the location of the function if nothing that came before it
+// set a code location first. This is useful, for example, if you want to
+// provide a default code location value to be used but not pay the overhead
+// of resolving that location until it's clear that you will need to. This
+// should appear at the end of a TraceOption list (or at least before any
```

Comment:
I think you meant _after_ any other options that want to specify the code location

Reply:
Er, yes. _after_.